addons/l10n_it_edi/wizard/account_move_send.py

```diff
@@ -1,12 +1,221 @@
 # -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import models
+from base64 import b64encode
+from markupsafe import Markup
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+from odoo.addons.l10n_it_edi.tools.xml_utils import format_errors
+from odoo.addons.account_edi_proxy_client.models.account_edi_proxy_user import AccountEdiProxyError
 
 
 class AccountMoveSend(models.Model):
     _inherit = 'account.move.send'
 
-    def _get_mail_attachment_from_doc(self, doc):
-        if doc.edi_format_id.code == 'fattura_pa':
+    enable_l10n_it_edi = fields.Boolean(compute='_compute_enable_l10n_it_edi')
+    l10n_it_edi_checkbox = fields.Boolean(
+        'Tax Agency (Italy)',
+        compute='_compute_l10n_it_edi_checkbox',
+        store=True,
+        readonly=False,
+        help=(
+            "Send the invoice to the Italian Tax Agency.\n"
+            "It is set as readonly if a report has already been created, to avoid inconsistencies.\n"
+            "To re-enable it, delete the PDF attachment."
+        )
+    )
+    l10n_it_edi_readonly = fields.Boolean(compute='_compute_l10n_it_edi_readonly')
+    l10n_it_edi_proxy_user_registered = fields.Boolean(string='Is IT EDI Proxy User Registered?', compute='_compute_l10n_it_edi_proxy_user_registered')
```

Comment:
This field isn't needed anymore, right?

Reply:
You mean, as a field? Yes, not needed 😅 👍🏻 
![image](https://github.com/odoo/odoo/assets/1665365/b2021733-03c3-41e4-93e2-6b7c77596f07)